.travis.yml

```diff
@@ -22,12 +22,17 @@ script:
 - py.test --cov --ignore=test/
 - pip install -r requirements.test.txt
 - ${DIST_REPO}/scripts/package.sh "redhat" "7"
+- ${DIST_REPO}/scripts/package.sh "ubuntu" "14.04"
+- sudo chown -R travis:travis ${DIST_REPO}/rpms/build
+- sudo chown  travis:travis ${DIST_REPO}/deb_dist/*.deb
 before_deploy: PKG_VERSION=$(python -c "import icontrol; print(icontrol.__version__)")
 deploy:
   - provider: releases
     api_key:
       secure: Nc4NrVZqv5L0Ss8UQh0Glx8AKkL1NfClcvmOrg6kyj+jqnZYbqVbWN7Od8K9tp9pD613mQ86RL8vDV25wxEQY/Z0MidKuWqmM7PkriCe6ZXYhO2qB/jTouWr6ucv2xz4CZE2HRV3gJMKT6zFzKUZD2zUcojoIfzOM1KK2ggohEzkXXzpCtXFbVRB5B5WpMJ5+MQKBKGAUIF2RCiMiVRkqIfEg9dVziNNsSfXOjq5zelEIx3ePj/9/1OCrcjIpdp1SDc7soM79JDcmTyUkeUiczrZyXqw9972wI3zYDTAK/cDyPv/DzHF+N4jLOlI8j1lx8u/tgukAN98x2PqLhkxnezz1wGX17UDyQEvhCXs94+dhB20QulVfjCz+t2xaFozsC5z2C22ogRNIWtIg65x/Uj8YyO1AqNG8gmEYHErlQQGUcdyWhoApvAEF7BGFRZzlyWRBpxu3m4LKPRhahF1g3qOqVt3SHNG2uF4zJOJf+8MBG07A0QNhWCfre5sszsxUaiaePjUrABAgRijU2MPnOwaEzfnPoFH6j40CrpYhVQRn1v97e8Nk547gErIaAoATeJ4XDhAOXWj8qZrN9XBRtnu0AqVL59JtpeORAT5PWjM2bU5gUYk6CuBofdVOVo7qBWFL0nwu8hUneD5ur6t2v6TinktW4zheNNHQUFsH/4=
-    file: ${DIST_REPO}/rpms/build/f5-icontrol-rest-${PKG_VERSION}-1.el7.noarch.rpm
+    file:
+      ${DIST_REPO}/rpms/build/f5-icontrol-rest-${PKG_VERSION}-1.el7.noarch.rpm
+      ${DIST_REPO}/deb_dist/python-f5-icontrol-rest_${PKG_VERSION}-1_1404_all.deb
```

Comment:
Is 1404 standard convention to reflect the distro explicitly in the name, or 'trusty'?